portfolio/src/App.css

```diff
@@ -1,5 +1,12 @@
+/* https://coolors.co/05668d-427aa1-ebf2fa-679436-a5be00 - Color Palette */
+/* 131112 - Smoky Black, FFE74C - Minion Yellow, 7DC95E - Mantis
+EDAE49 - Sunray, F9C80E - Jonquil
```

Comment:
Instead of writing them down, add them as CSS variables. https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties